@utahdts/utah-design-system-header/src/js/renderables/mainMenu/renderMainMenu.js

```diff
@@ -120,9 +120,10 @@ export function renderMainMenu() {
         // if have both an action url and menu items, show a page link since the menu can't be both clicked to open
         // the sub menu AND clicked to got to the link
         if (
-          menuItem.actionFunction
+          (menuItem.actionFunction
```

Comment:
...not related to this line...
Mobile Main menu still shows the overview item, so need to update the mobile menu rendering to also skip the overview.

Reply:
Embarrassing...